### ISSUE TITLE
Make alternative to hamburger menu

### ISSUE THREAD

**Issue by juliusosokinas** [ExCiteS/geokey#364]
T3a - Funka Accessibility Review.

At least a text for screen readers should be provided. Ideally the menu item should be avoided - https://lmjabreu.com/post/why-and-how-to-avoid-hamburger-menus/.
